mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -369,9 +369,9 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
         Log.d(TAG, "onBackPressed()")
         if (controller.canGoBack()) {
             controller.goBack()
-        } else if (!isFullscreenEnabled) {
-            // Only handle back action in non-fullscreen mode, as we don't want to exit
-            // the app via back button in fullscreen mode
+        } else if (isFullscreenEnabled && lastSnackbar?.isShown != true) {
+            showSnackbar(R.string.press_back_to_exit)
```

Comment:
Can one attach tags to SnackBars? Would be a good application here, as right now, when pressing back in the wrong moment (e.g. connection change), the app will exit immediately.

Reply:
I'm not aware of that. I also tried to get and compare the snackbar text, no success.

Reply:
Maybe via `lastSnackbar.getView()`? Needs investigation, I'm not sure whether that returns the container or the actual view.

Reply:
One can attach a tag to `lastSnackbar.view`.